pkg/terminal/command.go

```diff
@@ -1454,6 +1474,20 @@ func continueUntilCompleteNext(t *Term, state *api.DebuggerState, op string, sho
 	}
 }
 
+func promptAutoContinue(t *Term, op string) (string, error) {
+	for {
+		answer, err := t.line.Prompt(fmt.Sprintf("[c] continue [s] stop her and cancel %s, [f] finish %s skipping all breakpoints? ", op, op))
```

Comment:
s/her/here/

Reply:
It is not clear that "continue" will hit the internal breakpoint (unless it counters others along the way). Maybe "continue %s hitting all breakpoints"?

Reply:
> s/her/here/

Done.

> It is not clear that "continue" will hit the internal breakpoint (unless it counters others along the way). Maybe "continue %s hitting all breakpoints"?

That makes it sound like it does the same thing as 'finish' IMO. It would have to be "continue and try to finish %s but repeat this prompt if a breakpoint is hit" but I think that's too verbose. I think just saying 'continue' here is fine, there's three options and with the other two it should be immediately obvious whether they are right or wrong, it should be clear by exclusion if you want 'continue' and it's a safe option to experiment with.

Reply:
I agree with @aarzilli here that the wording as is should be ok for users. If we get reports on confusion from a UX perspective we can reevaluate.

Reply:
That's fair.
Also, this could be explained in more detail in the documentation without polluting the prompt.